### DISCUSSION TITLE
Max-width breakpoints are not work as expected

### DISCUSSION THREAD

**Discussion by helluuuuu** [tailwindlabs/tailwindcss#9198] [Help]
**What version of Tailwind CSS are you using?**
3.1.8

**What build tool (or framework if it abstracts the build tool) are you using?**
Next.js 12.2.5

**What version of Node.js are you using?**
v14.17.3

**What browser are you using?**
Chrome

**What operating system are you using?**
macOS

**Reproduction URL**
https://play.tailwindcss.com/WmlTfmsWSe

**Describe your issue**

I am trying to setup max-width breakpoints according to [the article](https://tailwindcss.com/docs/screens#max-width-breakpoints). After that all of the breakpoints related classes are not working with the error 
**The `max-w-screen-xl` class does not exist. If `max-w-screen-xl` is a custom class, make sure it is defined within a `@layer` directive.**

Please correct me if I am doing something wrong.

**Comment by RobinMalfait** [comment]
You overrode the `screens` in your config and the `xl` value doesn't in there. If you just drop the screens from your config, then it works as expected: https://play.tailwindcss.com/Hg0jwCTdRA?file=config
